docs/src/main/asciidoc/hibernate-orm.adoc

```diff
@@ -98,11 +98,11 @@ They will often map to Hibernate ORM configuration properties but could have dif
 
 Also, Quarkus will set many Hibernate ORM configuration settings automatically, and will often use more modern defaults.
 
-Please see below section <<hibernate-configuration-properties, Hibernate ORM configuration properties>> for the list of properties you can set in `{config-file}`.
+Please see below section xref:hibernate-configuration-properties[Hibernate ORM configuration properties] for the list of properties you can set in `{config-file}`.
```

Comment:
I know it's outside scope, but this sentence needs fixing:

```suggestion
For a list of the items that you can set in `{config-file}`, see xref:hibernate-configuration-properties[Hibernate ORM configuration properties].
```

Reply:
It definitively does! :D I am on it!